spec/feature_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'takeaway'
+
+describe 'User Stories' do 
+
+  it 'So I can check if I want to order something, I would like to see a list of dishes'do
```

Comment:
Metrics/LineLength: Line is too long. [90/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)